profiles/arch/riscv/package.use.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 2019-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Alex Fan <alexfanqi@yahoo.com> (2021-07-13)
+# dependencies not keyworded/ use flag not tested
+net-analyzer/net-snmp lm-sensors rpm smux
+
```

Comment:
strictly, I'd prefer p.use.mask one package first, then keyword it
so this commit should go before keywording net-analyzer/net-snmp

Reply:
Thanks! I have reordered them